src/_data/sidenav.yml

```
@@ -754,6 +771,8 @@
       permalink: /resources/courses
     - title: Learn Dart
       permalink: /resources/bootstrap-into-dart
+    - title: Support
```

Can you make this a task-based title like the surrounding entries? "Get support", "Get help", or similar.

```suggestion
    - title: Get support
```